src/codeu/chat/client/core/UserContext.java

```diff
@@ -42,6 +43,10 @@ public ConversationContext start(String name) {
         new ConversationContext(user, conversation, view, controller);
   }
 
+  public UserInterest begin(String name) {
+    return controller.newUserInterest(name);
```

Comment:
add parameter to pass in `user.id`, similar to how `newConversation` is called above on line 40.